django/forms/renderers.py

```diff
@@ -20,6 +20,11 @@ class BaseRenderer:
     form_template_name = "django/forms/div.html"
     formset_template_name = "django/forms/formsets/div.html"
     field_template_name = "django/forms/field.html"
+    bound_field_class = "django.forms.boundfield.BoundField"
+
+    # TODO workaround for circular import / resolve reference issue.
+    def get_bound_field_class(self):
+        return import_string(self.bound_field_class)
```

Comment:
An alternative to this is that this could be a property. This would avoid the need for the `get_bound_field_class()` method. 

```suggestion

    @property
    def bound_field_class(self):
        from django.forms.boundfield import BoundField

        return BoundField
```

This allows setting a custom `BoundField` on the `FORM_RENDERER` class like this:

``` Python
class BoundFieldWithoutColon(BoundField):
    def label_tag(self, contents=None, attrs=None, label_suffix=None, tag=None):
        return super().label_tag(contents=contents, attrs=attrs, label_suffix="", tag=None)


class CustomRenderer2(DjangoTemplates):
    bound_field_class = BoundFieldWithoutColon
```

What do you think?

Reply:
Yes, looks better.